### ISSUE TITLE
Elevated circulating alpha-aminobutyric acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6049]
**Preferred term label:** Elevated circulating alpha-aminobutyric acid concentration

**Synonyms** Elevated circulating 2-aminobutyric acid concentration

**Definition (free text, please give PubMed ID)** Concentration of alpha-aminobutyric acid in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormality of serum amino acid level HP:0003112

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033110
